### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by morganizwd**

### CODE REVIEW

**Comment by morganizwd** [comment]
[Задания №1](https://trello.com/c/wR9ezMRF/64-%D0%B7%D0%B0%D0%B4%D0%B0%D0%BD%D0%B8%D1%8F-%E2%84%961)

File: Task3/Task3/Program.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                Console.WriteLine("ЭТО НЕ ЧИСЛО!");
+                goto jebnytiy;
+            }
+
+
+
+
+
+
+
```
Comment (PaShkadun) [inline Task3/Task3/Program.cs:26]: Remove.

File: Task3/Task3/Program.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace ConsoleApp1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+        jebnytiy:
```
Comment (PaShkadun) [inline Task3/Task3/Program.cs:9]: Don't use GOTO.

File: Task3/Task3/Program.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            Console.WriteLine("Введите число: ");
+            try
```
Comment (PaShkadun) [inline Task3/Task3/Program.cs:11]: Don't use try-catch.

**Pull request merged**